File: src/main/java/org/zwobble/mammoth/internal/util/Paths.java
package org.zwobble.mammoth.internal.util;

public class Paths {
    private Paths() {}

    public static String getExtension(String path) {
        int dotIndex = path.lastIndexOf('.');
        return dotIndex == -1 ? "" : path.substring(dotIndex + 1);
    }
}

File: dotnet/Mammoth/Couscous/mammoth/internal/util/Paths.cs
namespace Mammoth.Couscous.org.zwobble.mammoth.@internal.util {
    internal static class Paths {
        internal static string getExtension(string path) {
            int dotIndex = path.LastIndexOf('.');
            return dotIndex == -1 ? "" : path.Substring(dotIndex + 1);
        }
    }
}
